core/planner.c

```diff
@@ -34,6 +34,7 @@ int decostoplevels_imperial[] = { 0, 3048, 6096, 9144, 12192, 15240, 18288, 2133
 				325120, 345440, 365760, 386080 };
 
 double plangflow, plangfhigh;
+int bottom_time = 0;
```

Comment:
Wouldn't it make sense to add this to struct deco_state? I am trying to weed out global variables in the planner so it can run multi-threaded (which currently it cannot but gives garbage) as that is what we need for a snappier UI.